front/ProfilePage.tsx

```diff
@@ -55,8 +55,8 @@ const ProfileHoc = (tab) => {
                     alt="User's profile image"
```

Comment:
Please make sure to clear all data from the database before the test, use faker and custom methods to generate fake data, and use the PageObject pattern for tests as mentioned in the task description.

Reply:
done